app/cdash/public/headeradminproject.xsl

```diff
@@ -0,0 +1,69 @@
+<xsl:stylesheet
+    xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version='1.0'>
+
+    <xsl:output method="xml" doctype-public="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd" encoding="UTF-8"/>
+    <xsl:template name="headeradminproject">
+
+<div id="header">
+ <div id="headertop">
+  <div id="datetime">
```

Comment:
We used to show links to "All Dashboards" and "My CDash" here. I presume these are handled elsewhere now?